api/src/main/java/com/datastrato/gravitino/authorization/SecurableObjects.java

```diff
@@ -18,6 +20,19 @@ public class SecurableObjects {
 
   private static final Splitter DOT_SPLITTER = Splitter.on('.');
 
+  /**
+   * Create the securable object {@link SecurableObject} with the given object namespace.
+   *
+   * @param type The type of the object
+   * @param namespace The object
+   * @param privileges The privileges of the securable object
+   * @return The created securable object {@link SecurableObject}
+   */
+  public static SecurableObject ofNamespace(
```

Comment:
I think it is not proper to use `Namespace` here, because `Namespace` is typically referring to a parent name, like namespace of a table is "catalog.schema", so it is ambiguous to use `Namespace` here, as we don't know whether it should be a Namespace or `NameIdentifier` here.

I would suggest to make `of` public, or you can use `parse`.